xla/service/latency_hiding_scheduler.cc

```diff
@@ -2760,6 +2760,8 @@ absl::StatusOr<bool> LatencyHidingScheduler::Run(
   LOG(INFO) << "LatencyHidingScheduler current memory usage: "
             << scheduler_core_->GetMemoryPeak()
             << " bytes. Current limit: " << scheduler_core_->GetMemoryLimit();
+  SetKVMetric(module, std::string(kPeakMemoryUsage),
```

Comment:
If you're using HloPassPipeline primitives can you explicitly use the `hlo_pass_pipeline` include then?